src/test/fidelity/artifact-creator.ts

```diff
@@ -189,7 +189,20 @@ export class ArtifactCreator {
           });
         });
 
-        await Promise.all([modelLoads, environmentChanges]);
+        const modelVisible = new Promise((resolve, reject) => {
+          const timeout = setTimeout(reject, 10000);
+
+          modelViewer.addEventListener('model-visibility', (event) => {
```

Comment:
I think we can just replace the `environment-change` event promise with this one. `model-visibility` will only be fired once the model is fully rendered and ready to be revealed, so it implies `environment-change`.

Reply:
Sure thing, fixed.